docker/entry.sh

```diff
@@ -6,9 +6,16 @@ echo $@
 EXTUID=`stat -c %u /build`
 EXTGID=`stat -c %g /build`
 
-# Change 'build' UID in the container to match the owner of the
+# Create 'build' user in the container to match the owner of the
 # build directory, so that built packages will have the correct
 # owner outside the container.
-usermod --non-unique -u $EXTUID build
-groupmod --non-unique -g $EXTGID build
+groupadd build --gid $EXTGID      \
+               --non-unique
+useradd build --groups mock,wheel \
+              --home-dir /build   \
+              --uid $EXTUID       \
+              --gid $EXTGID       \
+              --no-create-home    \
+              --non-unique
+
```

Comment:
I did literally mean just the containing folder and not a recursive but so long as that gets the correct permissions as a result of the map then all is good.